src/ConfigProvider.php

```diff
@@ -159,23 +161,25 @@ public function getViewHelperConfig()
     {
         return [
             'aliases'   => [
-                'countryCodeDataList' => View\Helper\CountryCodeDataList::class,
-                'currencyformat'      => View\Helper\CurrencyFormat::class,
-                'currencyFormat'      => View\Helper\CurrencyFormat::class,
-                'CurrencyFormat'      => View\Helper\CurrencyFormat::class,
-                'dateformat'          => View\Helper\DateFormat::class,
-                'dateFormat'          => View\Helper\DateFormat::class,
-                'DateFormat'          => View\Helper\DateFormat::class,
-                'numberformat'        => View\Helper\NumberFormat::class,
-                'numberFormat'        => View\Helper\NumberFormat::class,
-                'NumberFormat'        => View\Helper\NumberFormat::class,
-                'plural'              => View\Helper\Plural::class,
-                'Plural'              => View\Helper\Plural::class,
-                'translate'           => View\Helper\Translate::class,
-                'Translate'           => View\Helper\Translate::class,
-                'translateplural'     => View\Helper\TranslatePlural::class,
-                'translatePlural'     => View\Helper\TranslatePlural::class,
-                'TranslatePlural'     => View\Helper\TranslatePlural::class,
+                'countryCodeDataList'       => View\Helper\CountryCodeDataList::class,
+                'currencyformat'            => View\Helper\CurrencyFormat::class,
+                'currencyFormat'            => View\Helper\CurrencyFormat::class,
+                'CurrencyFormat'            => View\Helper\CurrencyFormat::class,
+                'dateformat'                => View\Helper\DateFormat::class,
+                'dateFormat'                => View\Helper\DateFormat::class,
+                'DateFormat'                => View\Helper\DateFormat::class,
+                'numberformat'              => View\Helper\NumberFormat::class,
+                'numberFormat'              => View\Helper\NumberFormat::class,
+                'NumberFormat'              => View\Helper\NumberFormat::class,
+                'plural'                    => View\Helper\Plural::class,
+                'Plural'                    => View\Helper\Plural::class,
+                'translate'                 => View\Helper\Translate::class,
+                'Translate'                 => View\Helper\Translate::class,
+                'translateplural'           => View\Helper\TranslatePlural::class,
+                'translatePlural'           => View\Helper\TranslatePlural::class,
+                'TranslatePlural'           => View\Helper\TranslatePlural::class,
+                'translateWithParams'       => View\Helper\TranslateWithParams::class,
+                'translatePluralWithParams' => View\Helper\TranslatePluralWithParams::class,
```

Comment:
The new view helpers should be used if the call is made with `translate` or `translatePlural` in a view script.

```suggestion
                'translate'                 => View\Helper\TranslateWithParams::class,
                'Translate'                 => View\Helper\TranslateWithParams::class,
                'translateplural'           => View\Helper\TranslatePluralWithParams::class,
                'translatePlural'           => View\Helper\TranslatePluralWithParams::class,
                'TranslatePlural'           => View\Helper\TranslatePluralWithParams::class,
```
As the new view helpers only add a new optional parameter, the user does not have to change anything and everything works as before.